tests/conftest.py

```diff
@@ -1,6 +1,17 @@
-# import pytest
+from importlib import import_module
 
+from django.conf import settings as django_settings
 
-# @pytest.fixture
-# def some_fixture(request):
-#     return "foo"
+import pytest
+
+
+@pytest.fixture
+def cache_session_store(settings):
+    settings.SESSION_ENGINE = "django.contrib.sessions.backends.cache"
```

Comment:
Are these settings set directly because they are ran outside of a testcase provided by django (e.g `django.test.TestCase`)?

Reply:
The db session is not really necessary because its set in the settings.

I've removed both fixtures and replaced them with cache override in the cached version